lib/rubocop/cop/lint/block_alignment.rb

```diff
@@ -7,13 +7,41 @@ module Lint
       # This cop checks whether the end keywords are aligned properly for do
       # end blocks.
       #
+      # Three modes are supported through the `AlignWith` configuration
+      # parameter:
+      #
+      # If it's set to `start_of_block` the `end` shall be aligned with the
+      # start of the line where the `do` appeared.
+      #
+      # If it's set to `start_of_line`, the `end` shall be aligned with the
+      # start of the line where the expression started.
+      #
+      # If it's set to `everything` (which is the default), the `end` is
+      # allowed to be in either location. The autofixer will default to
+      # `start_of_line` as that was the default before the options were added.
```

Comment:
Don't mention what the default was before options were added. Just succinctly and concisely explain how it works now.